ReactSkia/sdk/OSKConfig.h

```diff
@@ -11,13 +11,9 @@ namespace rns {
 namespace sdk {
 
 // Dark Theme configuration
-#define OSK_DARK_THEME_BACKGROUND_COLOR       SkColorSetARGB(0xFF, 0x34, 0x34, 0x34) //Dark Gray
```

Comment:
Moved common style configuration between subwindow [Alert & OSK] to seperate file ["sdkStyleConfig"]